Plug-Ins/NovaTools/Galaxy Editor/SystemView.swift

```diff
@@ -52,6 +54,10 @@ class SystemView: NSView {
         links = try (0..<16).map { _ in
             Int(try reader.read() as Int16)
         }
+        navDefaults = try (0..<16).map { _ in
+            Int(try reader.read() as Int16)
+        }
+        hasStellars = navDefaults.contains(where: { $0 > -1 })
```

Comment:
Might be best to make this >= 128. Sometimes unused values can get left at 0 rather than -1.

Reply:
Makes sense. For good measure I've made it compare against the valid ID range of 128-2175 inclusive: https://github.com/andrews05/ResForge/pull/18/commits/02d531a8cf651fa0448ca7f460d1ebab4e0f6c95#diff-4ab8513ab7cc9b86c943d689d46816ecfc649f887d786e6feb059ac6a3d721d5L60